climt/_core/initialization.py

```diff
@@ -92,6 +92,8 @@ def init_ozone(array_dims, quantity_description, initial_state):
     if array_dims[-1] == 30:
         target_profile = profile
 
+    target_profile[-1] /= 10
```

Comment:
Should probably add a comment saying why you're doing this.

Reply:
done